### DISCUSSION TITLE
Keeping a fix npm version whith renovate

### DISCUSSION THREAD

**Discussion by Demha00** [renovatebot/renovate#21422] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

gitlab.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Hello,

i would like to know if there's a way to fix npm version (to 9.5.0) (with node 18.15.0) and not let renovate_bots to update it (and merge it) to a higer version.
i can't see where in the renovate.json file i can do this.

Below is what i've tried so far and it didn't worked.

Best regards.

npm ERR! code EBADENGINE
npm ERR! engine Unsupported engine
npm ERR! engine Not compatible with your version of node/npm
npm ERR! notsup Not compatible with your version of node/npm
npm ERR! notsup Required: {"node":"18.15.5","npm":"9.6.4"}
npm ERR! notsup Actual:   {"npm":"7.20.5","node":"v14.17.0"}

**Comment by viceice** [comment]
don't use `renovate/node`, it's deprecated and no longer updated.


renovate by default does not merge any updates. it's up to you to configure package rules for that.

you can also use package rules to disable specific updates. there are samples inside the docs.

**Reply by Demha00** [reply]
Thanks for the answer.

"don't use renovate/node, it's deprecated and no longer updated"

Is there a source or a doc where it's says (or a way i can see) it's deprecated ? If you can past me a link that would be great.

**Reply by viceice** [reply]
checkout the source repo, it's archived. search for `docker-node` inside this organization.

you can use `containerbase/node` as replacement. renovate should already suggest that change

**Reply by Demha00** [reply]
Thank you @viceice

**Comment by rarkins** [comment, marked as answer]
ignoreDeps=npm

**Reply by Demha00** [reply]
Thank you, it worked.
